### DISCUSSION TITLE
adding nonce to the application for CSP

### DISCUSSION THREAD

**Discussion by pcodiant** [opcodesio/log-viewer#165] [Ideas]
As we when we are adding secure headers in the application it is blocking the entire screen to render.

Is there any way to add nonce in the script to handle the application? So that we can display the log file without losing the security of the application

**Comment by arukompas** [comment]
Hey @pcodiant , I'm not sure I understand the issue.

Could you elaborate on the problem with screenshots and/or code snippets? What is blocking the Log Viewer from being rendered?
